stable/joomla/README.md

```diff
@@ -0,0 +1,97 @@
+# Joomla!
+
+[Joomla!](http://www.joomla.org/) is a PHP content management system (CMS) for publishing web content. It includes features such as page caching, RSS feeds, printable versions of pages, news flashes, blogs, search, and support for language international.
+
+## TL;DR;
+
+```console
+$ helm install stable/joomla
```

Comment:
Not from the stable repository, the CI pulls the dependency before packaging it.